netadapter/router/route.go

```diff
@@ -101,3 +101,8 @@ func (r *Route) Close() {
 	r.closed = true
 	close(r.channel)
 }
+
+// IsEmpty returns whether the route has any pending messages
```

Comment:
I prefer the format "IsEmpty returns true/false if..."

Usually the format "XXX returns whether YYY" implies XXX returns true if YYY.
In this case it's the other way around.

"XXX returns YYY if ZZZ" is doesn't have this ambiguity.